pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1893,6 +1893,21 @@ class Timedelta(_Timedelta):
         Values for construction in compat with datetime.timedelta.
         Numpy ints and floats will be coerced to python ints and floats.
 
+    Attributes
+    ----------
+    resolution: Timedelta
+        Represents the smallest difference between two time units that can be
+        represented by the Timedelta object.
+        Fixed at 0 days 00:00:00.000000001 (1 nanosecond).
```

Comment:
I think this is an example? I'd suggest removing this - we don't give examples in the attribute descriptions and it doesn't seem necessary.

Same applies to the two attributes below.